### ISSUE TITLE
Pasting bone positions across skeletons messes up IK if they have different bindings

### ISSUE THREAD

**Issue by erikari** [EsotericSoftware/spine-editor#236]
Expected:
If all the bones are selected and pasted, Spine should consider the bending of a IK and change it if necessary

Result:
IK constraint gets wrecked. Bones have a strange angle.

Happens every time.

Repro steps:
1.open a project where there's a IK constraint.
2. import it again to have a second skeleton
3. move the second skeleton's bones and IK targets
4. change the bending direction of the IK
5. select all the bones and the IKs, copy them on world axis
6. deactivate the skeleton, reactivate the original skeleton
7. select all the bones and IKs of the original skeleton
8. paste, admire wreckage.

Before pasting, first skeleton:
![before-but with inverted ik](https://image.prntscr.com/image/FR09J3rbTYOt3uiaX_3JUg.png)

After pasting on second skeleton:
![after-pasting causes bone to be misplaced, also the mesh is FU](https://image.prntscr.com/image/tmptFDn-SE6tQjkdF9_kEg.png)

**Comment by NathanSweet** [comment]
We could do this for IK constraints, but there's not much we can do for a path or transform constraint. However, when pasting a pose to bones, it's expected that the bones are adjusted, but not that constraints are changed. That seems surprising.

@erikari When does this situation come up? I have a feeling it never makes sense to paste to bones that are constrained differently from what was copied.

**Comment by erikari** [comment]
@NathanSweet sometimes you want the bending of your IK to be flipped (e.g. you need to flip a foot) so checking the bending is ok.

Having a different path and wanting to paste its coordinates on something else seems a little crazy even in my opinion, 
regarding transform constraints, I think that as long as the settings stay the same but maybe you move the target bone, you should be able to paste, and in fact, (as far as I remember from doing those shenanigans) you can.

So basically the problem is the bending of a standard IK.

**Comment by NathanSweet** [comment]
3.7.43-beta

What we do is when pasting a bone transform we 1) don't adjust 2 bone IK parent bone rotation, and 2) don't adjust 2 bone IK child bone rotation or position. If the IK can be disabled in the tree (with the visibility dot) so the FK position is being modified, then the pasting happens without these restrictions.
